### DISCUSSION TITLE
Using gfs_quarter_deg_hourly_to_hourly_mean

### DISCUSSION THREAD

**Discussion by alisboainegi** [SolarArbiter/solarforecastarbiter-core#755] [Q&A]
I am trying to use the forecast of  solarforecastorbiter.

 Hello!

When I  run:
models.gfs_quarter_deg_hourly_to_hourly_mean(latitude, longitude, elevation, init_time, start, end, interval_label)

Where:
- latitude: - 41
- longitude: -8
- elevation: 104
- init_time: pd.Timestamp(datetime.date.today(), tz=self.tz)
- start: pd.Timestamp(datetime.date.today(), tz=self.tz)
- end: start + pd.Timedelta(days=2)
- interval_label: beggining
 
I get this error:
FileNotFoundError: gfs_0p25\2021\11\19\00\gfs_0p25.nc does not exist 

Could you help me understand my mistake?

**Comment by wholmgren** [comment]
Hi @alisboainegi. Have you downloaded the NWP data? The functions in `models.py` assume you already have that data. The easiest way to download data is to use the `solararbiter` [command line tool](https://solarforecastarbiter-core.readthedocs.io/en/latest/cli.html) with the `fetchnwp` argument. That's easy to do if you install the package using the conda forge package on linux, but harder on windows due to dependency issues. Or use the docker image.

Then it's a matter of pointing functions in `models.py` to the data you've downloaded. Those functions contain a keyword argument `load_forecast`. That must be a function that can load the forecast data in a specific format. By default that's `solarforecastarbiter.io.nwp.load_forecast`. You can wrap that function while defining a new `base_path` keyword argument or call `io.nwp.set_base_path` to change the directory.
